### ISSUE TITLE
Feature Request - Terms in object tracking search need definitions

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#4243]
**Is your feature request related to a problem? Please describe.**
"In Barcode" needs an explanation - actually all of the fields in object tracking do.

**Describe what you're trying to accomplish** 
Understand what is being searched in object tracking

**Describe the solution you'd like**
Provide links to the term definitions as we do in other search forms

**Describe alternatives you've considered**
Do nothing

**Additional context**
![image](https://user-images.githubusercontent.com/5725767/148819599-4500bbe7-0ee2-4669-9799-70920ddf5dc1.png)

**Priority**
Please assign a priority-label. Unprioritized issues gets sent into a black hole of despair.

**Comment by Jegelewicz** [comment]
@dustymc I thought this might be something I could do, but of course this form is set up different than others. Just looking at the first field

```
<label for="cat_num">Cat Num (comma-list OK)</label>
	<input type="text" name="cat_num" id="cat_num"  />
```

Would using the id in the label screw everything up?

This is how it works on the main Search page

```
<td class="lbl">
	<span class="helpLink" id="cat_num">Catalog&nbsp;Number:</span>
</td>
```

and I assume that id="cat_num" in the above is what brings up the handy pop-up with information on the search page.

![image](https://user-images.githubusercontent.com/5725767/151440565-872125ea-dce5-4d10-9883-b84ee70e9b89.png)

but I feel like it is doing something else in the object tracking search and if I move it, I'll break stuff.

If I know what to do here, I can fix this up with the links to definitions.

My first thought was this, but seems bad....

```
<span class="helpLink" id="cat_num">Catalog&nbsp;Number (comma-list OK):</span>
<input type="text" name="cat_num"  />
```

Does the input box need an id? Am I even making any sense? Could I do this instead?

```
<label for="cat_num">
         <span class="helpLink" id="cat_num">Catalog&nbsp;Number (comma-list OK):</span>
</label>
	<input type="text" name="cat_num"  />
```

I'd just play around, but I don't want to keep pestering you to pull stuff just to see what happens and I don't want to forget that I may have broken something that ends up in prod.

**Comment by dustymc** [comment]
There should be documentation in the code - use ``label for="xxx" class="helpLink" data-helplink="xxx"``

**Comment by Jegelewicz** [comment]
Where would I find that?

**Comment by Jegelewicz** [comment]
OK, well I just tried an experiment, so the next time you pull let me know and I'll see if it worked or not.

**Comment by dustymc** [comment]
Looks good. The code is in ajax.js. FYI I'm starting to bottle a release up, will probably grab a snapshot in about a half-hour.

**Comment by dustymc** [comment]
Oh - I clicked the button, it doesn't look so good after all. I fixed that one, duplicate IDs break the internet.

**Comment by Jegelewicz** [comment]
That's what I thought would happen!

**Comment by Jegelewicz** [comment]
OK - now that I have that as a template, I can work on the rest of them. THANKS!!!

**Comment by Jegelewicz** [comment]
Added links for all fields except Has Children.

**Comment by Jegelewicz** [comment]
Made updates to helplink for collection and container remark

**Comment by Jegelewicz** [comment]
@dustymc which one of these

![image](https://user-images.githubusercontent.com/5725767/151474847-c20d5068-b78b-4905-940a-a6a97c0f1658.png)

should go with the terms in Find Container?

![image](https://user-images.githubusercontent.com/5725767/151474914-092a281a-5b46-4f97-965e-58c36ef9be5f.png)

**Comment by Jegelewicz** [comment]
Well, this is like 95% done and maybe everyone just knows what install date is/should be.
